package.json

```diff
@@ -12,6 +12,13 @@
       "docs/*"
     ]
   },
+  "nyc": {
+    "exclude": [
+      "**/*.spec.js",
+      "src/lnd-actions/*",
```

Comment:
not sure I agree with excluding this, although I agree writing tests for this directory has not been a priority for us.

Reply:
OK. I will keep it simple

Reply:
This might be an OK addition as I think if we start rejecting PR on coverage than this will be a factor. I dont see us adding specs to those file as they are mainly a way for us to promisify the lnd calls